.github/workflows/ci.yml

```diff
@@ -101,4 +119,5 @@ jobs:
         if: contains( steps.getoutput.outputs.contents, 'FAILED (failures=' )
         run: |
           echo "${{ steps.getoutput.outputs.contents }}"
+          echo "There are failed tests"
```

Comment:
Why has this changed?

Reply:
this message is just to let user know there are some failed tests. I can remove it if you think it's unnecessary.

Reply:
Sorry, when I'd left the comment, it looked like line 121 was replaced (not augmented) with 122. Nvm.